tests/kernel/mem_heap/mheap_api_concept/src/test_mheap_concept.c

```diff
@@ -44,7 +54,16 @@ void test_mheap_malloc_align4(void)
 		k_free(block[i]);
 	}
 }
-
```

Comment:
This looks like there won't be an empty line between the previous function and the added comment, why this line was removed?